docs/blog/_posts/2021-06-25-scala301-rc2.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: blog-page
+title: Scala 3.0.1-RC2 – backports of critical bugfixes
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2021-06-25
+---
+
+This post is a quick announcement of Scala 3.0.1-RC2. This is the second release candidate for 3.0.1. The reason for this release is that a regression with respect to 3.0.0 was introduced by PR [#12519](https://github.com/lampepfl/dotty/pull/12519) which caused the compiler to fail where it shouldn't. We have fixed this regression in PR [#12827](https://github.com/lampepfl/dotty/pull/12827) and backported it to 3.0.1. This is the main reason for having 3.0.1-RC2 before 3.0.1 which is due in one week.
```

Comment:
According to https://github.com/lampepfl/dotty/milestones?direction=asc&sort=due_date, 3.0.1 is due for July 5 which is in two weeks, not one week

Reply:
It takes time to publish an article: this one will be out tomorrow,  Friday 25th. This effectively leaves 1 work week until 5th July.

Reply:
I see, but in the future I think for RCs release we should prioritize simple and fast release announcements (that can just list the backported PRs) over detailed blog posts that take many days to be published, after all we want people to test the RCs as soon as possible so we have time to react to their feedback before the final release.